common/list/lists-mws-en.html

```diff
@@ -0,0 +1,483 @@
+---
+{
+	"title": "Custom list styling using parent div element",
+	"language": "en",
+	"altLangPage": "lists-mws-fr.html",
+	"dateModified": "2024-04-04",
+	"description":"To accommodate managed web service product requirements regarding the lists, CSS classes must be applied to the parent element."
+}
+---
+<div class="wb-prettify all-pre"></div>
+<p>To accommodate managed web service product requirements regarding the lists, CSS classes must be applied to the parent element.</p>
+
+<div class="row">
+	<h2>Unstyled lists</h2>
+	<div class="col-md-6">
+		<h3>Custom markup</h3>
+		<div class="list-unstyled">
+			<ul>
+				<li>List item 1</li>
+				<li>List item 2
+					<ul>
+						<li>List item 2a</li>
+						<li>List item 2b</li>
+					</ul>
+				</li>
+				<li>List item 3</li>
+			</ul>
+		</div>
+		<p>Unstyled nested list:</p>
+		<ul>
+			<li>List item 1
+				<div class="lst-none">
+					<ul>
+						<li>List item 2a</li>
+						<li>List item 2b</li>
+					</ul>
+				</div>
+			</li>
+			<li>List item 2</li>
+			<li>List item 3</li>
+		</ul>
+		<h3>Code</h3>
+		<pre><code>//Unstyled list:
+&lt;div <strong>class=&quot;list-unstyled&quot;</strong>&gt;
+	&lt;ul&gt;
+		&lt;li&gt;...&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;
+//Unstyled nested list:
+&lt;ul&gt;
+	&lt;li&gt;List item 1
+		&lt;div <strong>class=&quot;lst-none&quot;</strong>&gt;
+			&lt;ul&gt;
+				&lt;li&gt;List item 1a&lt;/li&gt;
+				&lt;li&gt;List item 1b&lt;/li&gt;
+			&lt;/ul&gt;
+		&lt;/div&gt;
+	&lt;/li&gt;
+&lt;/ul&gt;
+</code></pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Default markup - <span class="label label-success mrgn-lft-sm">Recommended</span></h3>
+		<ul class="list-unstyled">
+			<li>List item 1</li>
+			<li>List item 2
+				<ul class="lst-none">
+					<li>List item 2a</li>
+					<li>List item 2b</li>
+				</ul>
+			</li>
+			<li>List item 3</li>
+		</ul>
+		<p>Unstyled nested list:</p>
+		<ul>
+			<li>List item 1
+				<ul class="lst-none">
+					<li>List item 2a</li>
+					<li>List item 2b</li>
+				</ul>
+			</li>
+			<li>List item 2</li>
+			<li>List item 3</li>
+		</ul>
+		<h3>Code</h3>
+		<pre><code>//Unstyled list:
+&lt;ul <strong>class=&quot;list-unstyled&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ul&gt;
+//Unstyled nested list:
+&lt;ul&gt;
+	&lt;li&gt;List item 1
+		&lt;ul <strong>class=&quot;lst-none&quot;</strong>&gt;
+			&lt;li&gt;List item 1a&lt;/li&gt;
+			&lt;li&gt;List item 1b&lt;/li&gt;
+		&lt;/ul&gt;
+	&lt;/li&gt;
+&lt;/ul&gt;
+</code></pre>
+	</div>
+</div>
+<div class="row">
+	<h2>Numbered lists</h2>
```

Comment:
Do we have a use case for this? That is simply the default styling.

Reply:
Done.